Library/Homebrew/dependable.rb

```diff
@@ -1,7 +1,7 @@
 require "options"
 
 module Dependable
-  RESERVED_TAGS = [:build, :optional, :recommended, :run, :linked]
+  RESERVED_TAGS = [:build, :optional, :recommended, :run]
```

Comment:
Decided to add it back to avoid breaking our DSL but have removed from homebrew/core and science.